x-pack/plugins/ml/public/components/influencers_list/influencers_list.js

```diff
@@ -24,6 +24,7 @@ import { FormattedMessage } from '@kbn/i18n/react';
 
 import { abbreviateWholeNumber } from 'plugins/ml/formatters/abbreviate_whole_number';
 import { getSeverity } from 'plugins/ml/../common/util/anomaly_utils';
+import { EntityCell } from '../anomalies_table/entity_cell';
```

Comment:
Similar to Pete's comment about the naming including `table`, maybe we can now move this to the general components folder now that it's not used in the anomalies table alone?

Reply:
Good one @walterra - the component has been moved to the general components dir and contains its own relevant styling. 👍 Done in 47d51d2810eb15b7497e6b5af8b5094b18db0280

Reply:
Great! Any idea why `entity_cell.js` shows up as deleted+created instead of moved in the git diff?

Reply:
Hm I created the `entity_cell` directory and then moved the file into it - perhaps because the dir did not exist before?